### ISSUE TITLE
Conversions are not renamed when renaming media.

### ISSUE THREAD

**Issue by kduma** [spatie/laravel-medialibrary#984]
When renaming media file, the file names of conversions aren't updated. Because links for conversions are generated using base file name, it breaks.

```PHP
$media = $model->addMedia('../file.jpg')->toMediaCollection();

dump($media->file_name); // file.jpg
dump($media->getUrl('thumb')); // /media/1/conversions/file-thumb.jpg

$media->file_name = 'new-name.jpg';
dump($media->getUrl('thumb')); // /media/1/conversions/new-name-thumb.jpg but in filesystem file is still named /media/1/conversions/file-thumb.jpg
```

Logic related with renaming is contained in MediaObserver.php file:
https://github.com/spatie/laravel-medialibrary/blob/23388b6f8191aa361ddc649de3688ca79a0606ab/src/MediaObserver.php#L15-L20

**Comment by brendt** [comment]
I'm going to look at this today, though it might be possible a fix won't be released until next week, depending on how easy it is to implement this fix.

**Comment by brendt** [comment]
This is fixed as of 7.1.0.

https://github.com/spatie/laravel-medialibrary/releases/tag/7.1.0
